includes/class-ajax-login.php

```diff
@@ -0,0 +1,374 @@
+<?php
+
+/**
+ * Ajax Login and Forgot password handler class
+ *
+ * @since 2.8
+ * @author Tareq Hasan <tareq@wedevs.com>
+ */
+
+class WPUF_Ajax_Login extends WP_Widget {
+
+    function __construct() {
+
+        parent::__construct(
+            'WPUF_Ajax_Login', 
+            __('WPUF Ajax Login', 'wpuf'), 
+            array( 'description' => __( 'Ajax Login widget for WP User Frontend', 'wpuf' ), ) 
+        );
+
+        add_action( 'widgets_init', array( $this, 'wpuf_ajax_login_widget' ) );
+        add_action( 'wp_enqueue_scripts', array( $this, 'enqueue_scripts' ) );

```

Comment:
Don't enqueue the script directly. Register first, then enqueue from inside the widget function.

When the widget isn't active, the script will not load.

Details: https://mikejolley.com/2013/12/02/sensible-script-enqueuing-shortcodes/